Runtime/Client/LootLockerServerApi.cs

```diff
@@ -131,14 +131,7 @@ IEnumerator coroutine()
                         text = webRequest.downloadHandler.text
                     };
 
-                    LootLockerErrorData errorData =
-                        LootLockerJson.DeserializeObject<LootLockerErrorData>(webRequest.downloadHandler.text);
-                    if (ErrorIsPreErrorCodes(errorData))
-                    {
-                        errorData = ExpandErrorInformation(webRequest.responseCode, webRequest.downloadHandler.text);
-                    }
-
-                    response.errorData = errorData;
+                    response.errorData = LootLockerJson.DeserializeObject<LootLockerErrorData>(webRequest.downloadHandler.text);
```

Comment:
This is the part I need explained to me before we go on with the change. If it does indeed work it's fantastic. So much cleaner. I know that I intentionally made it this cumbersome a couple of months ago because I hade edge cases where the respnse did not look like
```json
{
    "error_code": "asdasd",
    "error_doc_url": "asdkaskdl",
    ....
}
```
And if it doesn't then this will cause errorData to be null, which then will cause errorData to be null and the intended "helping" structure of errorData to be pointless. 

To summarize, I can approve this wholeheartedly as long as @ladydascalie can guarantee that **all** potential errors we can get (from Go and PHP, network issues, etc.) will be packaged in that beautiful structure. But I'll need to hear that case first.